content/en/blog/2023/how_to_debug_large_db_size_issue.md

```diff
@@ -121,6 +121,13 @@ and `dbSizeInUse`. Let's use the first endpoint as an example, its `dbSize` is 5
 ]
 ```
 
+etcd also exposes the following metrics,
```

Comment:
Optional suggestion to link throughtoourmain metrics docs with the content for getting started with etcd Prometheus metrics.

```suggestion
etcd also exposes the following [metrics](https://etcd.io/docs/v3.5/metrics),
```

Reply:
updated & thx